lib/entities.ts

```diff
@@ -345,7 +349,316 @@ export interface Order {
   filled_avg_price: string
   status: string
   extended_hours: boolean
-  legs: any
+  legs: RawOrder[]
```

Comment:
I'm not 100% sure if this works. The docs say that it's either null or a list of orders, but I don't know if the data is really identical. For example, I'm assuming you can only have a depth of 2 -- in other words, your legs can't have legs. If we want to enforce that, I'll need to do a bit more work here...